dds2val/tests/integration/feeder/test_ddsfeeder_int.py

```diff
@@ -0,0 +1,104 @@
+#################################################################################
+# Copyright (c) 2022 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License 2.0 which is available at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+#################################################################################
+
+import os
+import signal
+import threading
+import time
+import subprocess
+
+import pytest
+import sensor_msgs  # # noqa: F401
+import std_msgs
+from cyclonedds.domain import DomainParticipant
+from cyclonedds.pub import DataWriter
+from cyclonedds.topic import Topic
+from sdv.base import Config
+from sdv.vdb.client import VehicleDataBrokerClient
+
+from ....src import ddsfeeder as ddssrc
+
+
+async def run_dds_feeder_and_send_dds_msg(data, topic_name, topic_type):
+    # Run this after some delay, while dds feeder is running
+    def parallel_thread():
+        send_dds_messages(data, topic_name, topic_type)
+        time.sleep(1)
+        os.kill(os.getpid(), signal.SIGINT)
+
+    threading.Timer(5, parallel_thread).start()
+
+    # Run dds feeder until SIGINT signal via timer.
+    os.environ["VEHICLEDATABROKER_DAPR_APP_ID"] = "vehicledatabroker"
+    await ddssrc.main()
+
+
+@pytest.mark.asyncio
+async def test_ddsfeeder_start():
+    "Test case to verify data send on dds bus is received in broker"
+
+    # Use sdv SDK without dapr:
+    Config().disable_dapr()
+
+    databroker = subprocess.Popen(['docker', 'run', '--net=host', 'ghcr.io/eclipse/kuksa.val/databroker:master'], stdin=subprocess.PIPE,
+                     stdout=subprocess.PIPE)
+    # if it needs to pull the image it builds long
+    time.sleep(120)
+
+    # prepare vehicle data broker
+    vdb_client = VehicleDataBrokerClient()
+
+    await run_dds_feeder_and_send_dds_msg(
+        create_NavSatFix_dds_messages(["52.25,12.25,0"]),
+        "Nav_Sat_Fix",
+        sensor_msgs.msg.NavSatFix,
+    )
+
+    # Verify data received in broker is the same as send above
+    response = await vdb_client.GetDatapoints(["Vehicle.CurrentLocation.Latitude"])
```

Comment:
Here we use a client from Velocitas and in another file we specify that we shall use version v0.7.1. I see Velocitas as a downstream project from kuksa so having a dependency to Velocitas is not optimal as I see it. 

We only seem to use the Velocitas client to read a datapoint, so could we possibly manage without it? Possibly by adding a "get" method to https://github.com/boschglobal/kuksa.val.feeders/blob/feature/DDS_feeder/dds2val/src/databroker.py

Reply:
Sure, good point.

Reply:
Or maybe use kuksa client library here. Will change